### PR TITLE
Moved to modules/s3db/building.py

### PR DESCRIPTION
**Pull request by anurag-ks**
This is for my task - https://www.google-melange.com/gci/task/view/google/gci2014/5825114507575296

### CODE REVIEW

**Comment by anurag-ks** [comment]
@hitesh96db please review

File: modules/s3db/building.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
+# -*- coding: utf-8 -*-
+
+""" Sahana Eden Budget Model
```
Comment (hitesh96db) [inline modules/s3db/building.py]: Budget ?

**Comment by hitesh96db** [comment]
I get a ticket while saving a record, could you correct that as well ? and see if everything is working fine

**Comment by ramdesh** [comment]
Merging from @anurag-ks for GCI Task https://www.google-melange.com/gci/task/view/google/gci2014/5825114507575296

**Pull request merged**